k8s/values.yaml

```diff
@@ -0,0 +1,50 @@
+general:
```

Comment:
there should not be only one value file but per env value file

Reply:
@ggramal ok, done